coremltools/converters/nnssa/coreml/ssa_converter.py

```diff
@@ -2643,7 +2643,7 @@ def _convert_binary_elementwise(node):
                     raise ValueError("Incorrect configuration!! Alpha not provided for Elementwise Div operator")
                 alpha = 1 / float(alpha)
             elif op == 'sub':
-                if alpha and inputs[0] == input_names[0]:
```

Comment:
Can we add a simple unit test for this?

Reply:
@srikris 

Sure. I'd like to make a test code if you allow.
Do I add the test code, like `test_tf2_sub.py`, on [the test/neural_network foder](https://github.com/apple/coremltools/tree/master/coremltools/test/neural_network)?
If you have any good simple example for the unit test, please tell me. Actually this will be my first test code with python.

Reply:
Hi @tucan9389, since it's a conversion tests, can you add to this file: https://github.com/apple/coremltools/blob/master/coremltools/converters/tensorflow/test/test_convnets.py. Thanks for the patch!

Reply:
@1duo 
Thanks
I'm going to write a unit test with [the reference](https://github.com/apple/coremltools/blob/master/BUILD.md#running-unit-tests).

Reply:
@1duo 
I added the unit test code on `test_convnets.py`. But I cannot run the test on my local env because https://github.com/apple/coremltools/issues/663 . Do you have any advice on the issue?

Reply:
> @1duo
> I added the unit test code on `test_convnets.py`. But I cannot run the test on my local env because of #663 . Do you have any advice on the issue?

@srikris Do you have any insight for this build issue?